examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
No, but too much information is also not useful. We need to drastically cut down the number of examples we have because we dilute the information too much. I can add more text on the interpolation_methods.py example but if I have to do this for all examples we'll never have a gallery or even documentation that is in any way useful.